recipe/meta.yaml

```diff
@@ -90,11 +90,14 @@ outputs:
       run:
         - python >=3.7
         - grpcio >=1.47.0,<2.0dev
+        # the following is not needed here because grpcio <1.49.1 was not built
+        # with python 3.11 and we want to stay noarch: python
+        # grpcio >=1.49.1,<2.0dev  # [py>=311]
```

Comment:
Hmm, this turns out not to be true.  We've build older grcpio for python 3.11, for better or for worse...

It looks like we need to drop noarch: python for now :-(